draft-private-access-tokens.md

```diff
@@ -226,55 +227,55 @@ The Client sends the Issuer Name to the Mediator so the Mediator know where to
-that the Mediator derives is specific to a Policy Window, meaning that a CLIENT_ID
-will not map to the same ANON_CLIENT_ID after the Policy Window has elapsed.
+: The period over which an Issuer will track access policy, defined in terms of
+seconds and represented as a uint64. The state that the Mediator keeps for a Client
+is specific to a policy window. The effective policy window for a specific Client
+starts when the Clients first sends a request associated with an Issuer.
 
 ORIGIN_TOKEN_KEY:
 : The public key used when generating and verifying Private Access Tokens. Each
-Origin Token Key is unique to a single Origin.
+Origin Token Key is unique to a single Origin. The corresponding private key
+is held by the Issuer.
 
 ISSUER_NAME_KEY:
 : The public key used to encrypt the ORIGIN_NAME in request from Clients
 to the Issuer, so that Mediators cannot learn the ORIGIN_NAME value. Each
-Issuer Name Key is used across all requests on the Issuer, for different Origins.
+ISSUER_NAME_KEY is used across all requests on the Issuer, for different Origins.
 
 ORIGIN_NAME:
-: The Origin name identifies the Origin that requests and verifies Private Access Tokens.
+: The name of the Origin that requests and verifies Private Access Tokens.
 
 ANON_ORIGIN_ID:
-: The Anonymous Origin Identifier is generated by the Client and marked on requests
-to the Mediator and through to the Issuer.
-
-CLIENT_ID:
-: The Client Identifier represents a single client that has authenticated to the Mediator.
-The specifics of this identity are up to the Mediator, but it may be based on an
-attested device identifier or an account login that the Mediator can verify.
-
-ANON_CLIENT_ID:
-: The Anonymous Client Identifier is generated by the Mediator and used when forwarding
-requests to the Issuer. The Mediator maintains a mapping such that there is exactly one
-ANON_CLIENT_ID for each CLIENT_ID + ANON_ORIGIN_ID pair during a specific ISSUER_POLICY_WINDOW.
-
-ANON_ORIGIN_ID_PRIME:
-: The Anonymous Origin Identifier Prime is generated by the Mediator and sent to the Issuer,
-and corresponds to a unique ANON_ORIGIN_ID and a unique ANON_CLIENT_ID.
+: An identifier that is generated by the Client and marked on requests to the
+Mediator, which represents a specific Origin anonymously. The Client generates
+a stable ANON_ORIGIN_ID for each ORIGIN_NAME, to allow the Mediator to count
+token access without learning the ORIGIN_NAME.
+
+CLIENT_GROUP_ID:
+: An identifier chosen by the Mediator and sent to the Issuer, which represents
+a random cohort of Clients. Each Client is assigned a new CLIENT_GROUP_ID for each
+Issuer it uses every time a new policy window begins.
+
+GROUP_ORIGIN_ID:
+: An identifier chosen by the Issuer and sent to the Mediator, which corresponds
+to the tuple of an ORIGIN_NAME and a CLIENT_GROUP_ID. This identifier is used by
+the Mediator to ensure that Clients cannot create a mismatch between an
+ORIGIN_NAME and an ANON_ORIGIN_ID.
```

Comment:
The purpose of this should be to allow the Mediator and the Issuer to communicate about the Origin, including for the Mediator to enforce the Issuer's policy, right?

Reply:
It's just for the issuer to communicate to the mediator something about the origin that lets the mediator detect client's trying to abuse the system. The actual per-origin limit is enforce by the issuer.